raiden_contracts/contract_manager.py

```diff
@@ -211,14 +211,14 @@ def contracts_deployed_path(chain_id: int, version: Optional[str] = None):
     return data_path.joinpath(f'deployment_{chain_name}.json')
 
 
-def get_contracts_deployed(chain_id: int, version: Optional[str] = None):
+def get_contracts_deployed(chain_id: int, version: Optional[str] = None) -> Optional[Dict]:
     deployment_file_path = contracts_deployed_path(chain_id, version)
 
     try:
         with deployment_file_path.open() as deployment_file:
             deployment_data = json.load(deployment_file)
-    except (JSONDecodeError, UnicodeDecodeError) as ex:
-        raise ValueError(f'Cannot load deployment data file: {ex}') from ex
+    except (JSONDecodeError, UnicodeDecodeError, FileNotFoundError) as ex:
+        return None
```

Comment:
I still think we should throw an exception if `get_contracts_deployed` doesn't find a file for a supported version, because this might mean we did not release the package correctly. Currently only the last version + `pre_limits` is supported. 
For the rest (unsupported `chain_id`, `version`), it can return `None`.

Reply:
Dropped the second commit.